src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/UserManagement/UsersController.cs

```diff
@@ -0,0 +1,169 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Admin.ViewModels;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Validators.ValidationErrorMessages;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Validators.ValidationUtils;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api.UserManagement
+{
+    [Route("api")]
+    [ApiController]
+    [Authorize(Policy = "CanManageUsersOnSystem")]
+    public class UsersController : ControllerBase
+    {
+        private readonly IUserManagementService _userManagementService;
+
+        public UsersController(IUserManagementService userManagementService)
+        {
+            _userManagementService = userManagementService;
+        }
+
+        [HttpGet("user-management/users")]
+        [ProducesResponseType(200)]
+        [ProducesResponseType(404)]
+        public async Task<ActionResult<List<UserViewModel>>> GetUserList()
+        {
+            var users = await _userManagementService.ListAsync();
+
+            if (users.Any())
+            {
+                return Ok(users);
+            }
+
+            return NotFound();
+        }
+
+        [HttpGet("user-management/users/{userId}")]
+        [ProducesResponseType(200)]
+        [ProducesResponseType(404)]
+        public async Task<ActionResult<UserViewModel>> GetUser(string userId)
+        {
+            var user = await _userManagementService.GetAsync(userId);
```

Comment:
I would have changed the `_userManagementService.GetAsync` to use `IPersistenceHelper#CheckEntityExists` to get the user and change the signature of method `GetAsync` to return `Task<Either<ActionResult, UserViewModel>>`  because then this method could all be simplified to `return await _userManagementService.GetAsync(userId).HandleFailuresOrOk()`. The PersistenceHelper returns NotFound at the Service level.